### ISSUE TITLE
Is the "And" keyword supported ?

### ISSUE THREAD

**Issue by GregorD1** [TheBrainFamily/cypress-cucumber-preprocessor#31]
Hi - I am having trouble getting the "And" keyword to work - is it supported ? Thanks for any time.

**Comment by jbbn** [comment]
I had problems with this too.
For now I just repeating the word:

```gherkin
(...)
  When I fill the field "login" with "user"
  When I fill the field "password" with "secret"
(...)
```
I took a look in the source code and it doesn't seem too hard to implement an "And" ability.
I'm planning to do a PR in the next week.

**Comment by lgandecki** [comment]
@jbbn that shouldn't be hard to do and would be a great first contribution. :) If you don't figure it out I will do it next week. I need to also treat given/when/then/and all the same when matching, as other issue is pointing out. You can try that as well - it should make everything easier.

**Comment by GregorD1** [comment]
Thanks Guys, That gives me a workaround in the meantime.
I've been using the agile test management tool Hiptest to author the tests with my customer and export into SpecFlow syntax, there are various export options SpecFlow, Cucumber, CSharp.  Is there a specific syntax that you are targetting ?

**Comment by lgandecki** [comment]
Sorry @GregorD1 .. not sure what are you asking. We are using gherkin syntax that's used by cucumber.

**Comment by GregorD1** [comment]
Thanks - I had a doubt in my head after trying to get datatables working with my Cucumber export.....

  Scenario Outline: Define Recruitment Campaign Content (Panel Member Only) - Add Content (<hiptest-uid>)
    Given Recruiter navigates to CHS Community Portal
    And Recruiter has access to Campaign Management Module
    When Recruiter edits ${PageType}
    Then An image can be added to the content
    And An external link can be added to the content
    And A video can be added to the content
    And Sections can be re-ordered

    Examples:
      | PageType | hiptest-uid |
      | "Testimonial" | uid:275cfa33-5808-4884-bbb8-c7434ef8a398 |
      | "Case Study" | uid:9ce13ab8-3af8-4e63-811d-7b39265a63b2 |
      | "Area Support Team Page" | uid:26d9675b-26c1-48d5-b912-931328398384 |
      | "Who we are" | uid:0eb47818-0ba1-4d22-b3d1-98d2509a4c1b |
      | "How can you help" | uid:8188b24f-ba38-4ccf-8bea-a505b1d76073 |

It seemed that the package supports slightly different syntax for the parameter. This is what I got working with the package.....

Feature: RazorPagesMovie

I open the razor pages movie

Scenario Outline: Opening the razor pages movie
Given I open razor pages movie
When I click "<PageType>"
Then I see "<PageType>" in the title

Examples:
 | PageType | NoFill  |
 | About 	|		|
 | Contact 	|   	|

**Comment by GregorD1** [comment]
Ah! the angle brackets have been taken out by the editor above.

**Comment by lgandecki** [comment]
@GregorD1 I believe our syntax is based on this kind of syntax:
https://cucumber.io/docs/reference#scenario-outline

**Comment by GregorD1** [comment]
Thanks - I've re-authored my tests in the Hiptest tool and exported, it matches the syntax you have as per the reference. Thanks

**Comment by lgandecki** [comment]
This is fixed in 0.4.0 , please give it a try, and if still doesn't work as needed please create a new issue.
